streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/security/impl/AuthenticatorImpl.java

```diff
@@ -39,7 +40,7 @@ public class AuthenticatorImpl implements Authenticator {
   @Override
   public User authenticate(String username, String password) {
     User user = usersService.findByName(username);
-    if (user == null) {
+    if (user == null || user.getLoginType() == LoginType.LDAP) {
```

Comment:
If we add another login type,  will a bug occur?
```
if (user == null || user.getLoginType() != LoginType.PASSWORD) {
      return null;
  }
```

Reply:
You're right, I will fix it.